Examples/proj/Program.cs

```diff
@@ -124,11 +229,38 @@ public static void Main(string[] args)
     }
 }
 
-// 17 reports expected (18 with --pulse-increase-leak-recall flag)
+// 18 reports expected (19 with --pulse-increase-leak-recall flag)
 class InferResourceLeakTests
 {
     private static byte[] myBytes = new byte[] { 10, 4 };
 
+    /// <summary>
+    /// Validates that even though the Dispose occurs within a finally block, the exceptional
+    /// control flow is handled so that the analysis sees that the Dispose is not invoked. Also
+    /// gets coverage over finally blocks ending with throw instead of endfinally.
+    /// </summary>
+    public static void TryFinallyThrow()
```

Comment:
When I run infer on this example, no resource leak was reported. I was running the latest Infer main branch